extension/src/popup/components/accountHistory/NetworkInfo/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+import { getIconUrlFromIssuer } from "@shared/api/helpers/getIconUrlFromIssuer";
+
+import StellarLogo from "popup/assets/stellar-logo.png";
+
+import "./styles.scss";
+
+interface NetworkInfoProps {
+  assetIssuer: string;
+  assetCode: string;
+  assetType: string;
+  assetDomain: string;
+}
+
+export const NetworkInfo = ({
```

Comment:
I got confused for a minute thinking `NetworkInfo` was pulling data on Mainnet vs testnet before looking at the code. Thoughts on changing the name to something like `AssetNetworkInfo`, or something similar?